torch/csrc/export/example_upgraders.cpp

```diff
@@ -0,0 +1,89 @@
+#include <torch/csrc/export/example_upgraders.h>
```

Comment:
I still wish this file was in test/ but I guess if the build system is too annoying to figure out it's ok

Reply:
lol, it's indeed annoying